Library/Homebrew/download_strategy.rb

```diff
@@ -652,7 +652,7 @@ def update_submodules
 
   def checkout_submodules(dst)
     escaped_clone_path = cached_location.to_s.gsub(/\//, '\/')
-    sub_cmd = "git checkout-index -a -f --prefix=#{dst}/${toplevel/#{escaped_clone_path}/}/$path/"
+    sub_cmd = "git checkout-index -a -f --prefix=#{dst}/${toplevel/#{escaped_clone_path}/}/\'$path\'/"
```

Comment:
@jacknagel ok i'll make a pull request in a few hours... just in the middle of a meeting